scripts/verify-nupkgs.ps1

```diff
@@ -13,14 +13,14 @@ function Verify-Nuget-Packages($packageDirectory, $version)
     Write-Log "Starting Verify-Nuget-Packages."
     $expectedNumOfFiles = @{
         "Microsoft.CodeCoverage" = 57;
-        "Microsoft.NET.Test.Sdk" = 27;
-        "Microsoft.TestPlatform" = 623;
+        "Microsoft.NET.Test.Sdk" = 24;
+        "Microsoft.TestPlatform" = 599;
         "Microsoft.TestPlatform.Build" = 21;
-        "Microsoft.TestPlatform.CLI" = 426;
+        "Microsoft.TestPlatform.CLI" = 498;
         "Microsoft.TestPlatform.Extensions.TrxLogger" = 35;
-        "Microsoft.TestPlatform.ObjectModel" = 238;
+        "Microsoft.TestPlatform.ObjectModel" = 209;
         "Microsoft.TestPlatform.AdapterUtilities" = 62;
-        "Microsoft.TestPlatform.Portable" = 646;
+        "Microsoft.TestPlatform.Portable" = 598;
```

Comment:
All the smaller numbers make sense. We are removing the extra testhosts etc. But why CLI has more files? Is that because something brings more dependencies to support netstandard2.0?

Reply:
I confirm that this is all netstandard and system dlls:

![image](https://user-images.githubusercontent.com/11340282/179240187-fec0a404-9849-44c5-a9d9-33eaed47e23f.png)